background.ts

```diff
@@ -1228,6 +1236,29 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
       }
     }
   }
+
+  getBackendState(): BackendState {
+    return {
+      vmState: k8smanager.state,
+      locked,
```

Comment:
This should be the same as whatever the `locked` variable gets changed to ( `snapshotIsLocked` )